lizmap/modules/action/classes/actionConfig.class.php

```diff
@@ -61,15 +65,66 @@ public function __construct($repository, $project)
             return;
         }
 
+        // Convert old configuration (generated for LWC < 3.7)
+        if (is_object($this->config)) {
+            $this->convertOldConfig();
+        }
+
         // Get config
         $this->status = true;
     }
 
+    /**
+     * Convert an old "action" configuration (generated for LWC < 3.7)
+     * into the new format (array of actions instead of array of layers)
+     *
+     * @return array The new configuration
+     */
+    function convertOldConfig()
+    {
+        $config = $this->config;
```

Comment:
To invite people migrating to new version, and to remove this code from LWC later (3.9 ?) : 
* add a console.log to saying the format of the action is deprecated
* if the current user is an admin, add message bar asking for migrating ?
* raising a log in the administration panel ? (when loading the project from PHP side)

It's not only for us for the code maintenance, but also for them to make them aware about this new feature, new way etc.